_data/finance.yml

```diff
@@ -383,11 +383,12 @@ websites:
     facebook: VitalityUK
     img: vitality.png
 
-  - name: WaveApps
+  - name: Wave Accounting
     url: https://www.waveapps.com
     img: wave.png
     twitter: WaveHQ
     facebook: waveHQ
+    status: https://community.waveapps.com/discussion/3424/add-in-option-for-two-factor-authentication-2fa
```

Comment:
Do you see something which I don't? Because to me it looks like they don't have any intentions of adding 2FA let alone are actively working on it.

Reply:
![Screen Shot 2020-01-03 at 3 20 31 PM](https://user-images.githubusercontent.com/30846409/71754788-a470e900-2e3c-11ea-97df-ce0c78369585.png)

Reply:
I looked for a way to directly link to that specific comment, but it doesn't look possible.

Reply:
That's from March 14, Then in August and October they write this:
![](https://d.carlgo11.com/D5E0FCCCAA88F8/?p=bf8TmYjDgkeMH)

Reply:
Found a direct link, changing.

```suggestion
    status: https://community.waveapps.com/discussion/3424/add-in-option-for-two-factor-authentication-2fa#Comment_15005
```

Reply:
Oh, I didn't look that far. That's too bad. Sorry for the nuisance.